### ISSUE TITLE
Discrepancy between iiv_strategy setting and metadata.json information

### ISSUE THREAD

**Issue by schaedeli** [pharmpy/pharmr#70]
Hi
I just found a discrepancy between the iiv_strategy=3 in the run_modelsearch command, and the file metadata.json, where iiv_strategy=0. The modelsearch tool is actually adding IIV on the parameters of the transit compartment absorption parameters, so I guess the modelsearch tool is running as expected. Would you have a look into this?
Kind Regards, 
Franziska

```
# 1 - reduced_stepwise - iiv_strategy=3
# res_modelsearch_1 <- run_tool("modelsearch",
res_modelsearch_1 <- run_modelsearch( 
                            model=input_model,
                            search_space=sp1, 
#                            search_space=sp2, 
                            algorithm="reduced_stepwise", 
                            rankfunc="bic", 
                            iiv_strategy=3)

#summarize the results
```
metadata.json

```
{
    "tool_name": "modelsearch",
    "stats": {
        "start_time": "2022-05-25T12:34:50.960106+02:00"
    },
    "tool_options": {
        "search_space": "TRANSITS([1,3],*);PERIPHERALS(1..2);ABSORPTION(SEQ-ZO-FO);LAGTIME()",
        "algorithm": "reduced_stepwise",
        "iiv_strategy": 0,
        "rankfunc": "bic",
        "cutoff": null,
        "model": null
    },
    "common_options": {
        "dispatcher": "pharmpy.workflows.dispatchers.local_dask",
        "database": {
            "class": "LocalDirectoryToolDatabase",
            "toolname": "modelsearch",
            "path": "C:\\Users\\Public\\ADaMO\\Phase_1\\Model_11\\modelsearch\\modelsearch_dir1"
        }
    }
}
```

**Comment by stellabelin** [comment]
Hi Franziska!

Thanks for reporting this, I can confirm this is a bug! It works as expected in Python but not in R.

**Comment by joaoabr** [comment]
With another example:
```
res_iivsearch <- run_iivsearch(algorithm='brute_force',
                               model=best.struct.model,
                               iiv_strategy="fullblock",
                               rankfunc='bic',
                               cutoff=NULL)
```

is currently leading to:
```
    "tool_options": {
        "algorithm": "brute_force",
        "iiv_strategy": "no_add",
        "rankfunc": "bic",
        "cutoff": null,
        "model": null
    }
```

**Comment by stellabelin** [comment]
This should now be fixed (addressed in 67481bf4ae8be4baf3fa92dc17fc4f592d037309).
